src/Microsoft.AspNetCore.Antiforgery/Internal/AntiforgeryOptionsSetup.cs

```diff
@@ -28,7 +28,14 @@ public static void ConfigureOptions(AntiforgeryOptions options, DataProtectionOp
 
         private static string ComputeCookieName(string applicationId)
         {
-            using (var sha256 = SHA256.Create())
+            HashAlgorithm sha256;
+#if NETSTANDARD1_3
+            sha256 = SHA256.Create();
+#else
+            sha256 = new SHA256Cng();
```

Comment:
@PinpointTownes Thanks for the feedback! @blowdart is currently on vacation but once he returns I will run it across him.

@dougbu: May have been, this approach is what @blowdart recommended offline.